### PR TITLE
Addressing Issue 183: Changed From Flask Development Server to WSGI in Docker

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: multiscanner/distributed/api.py [hunk 1]
```diff
@@ -1065,7 +1066,16 @@ def _main():
         )
         ms_process.start()
 
-    app.run(host=api_config['api']['host'], port=api_config['api']['port'])
+    in_docker = os.getenv("PRODUCTION", False)
+    
+    if not in_docker:
```
Comment (emmanvg) [inline multiscanner/distributed/api.py:1071]: Note that if the environment variable `"PRODUCTION"` is not set. The return value from line 1069 `os.getenv("PRODUCTION", False)` is `False`. The current check here (L1071) will evaluate to `True` and be executed which I think is **not** the intended behavior.

Comment (emmanvg) [reply multiscanner/distributed/api.py:1071]: The other part to this comment and also seen in other places is that its only checking for the presence of the environment variable, but not checking its value. Thus `ENV PRODUCTION No` would still execute the portion of the code meant for production.

Comment (ghost) [reply multiscanner/distributed/api.py:1071]: Thank you for the comments- you are correct. Will fix the code to work as intended.

Comment (ghost) [reply multiscanner/distributed/api.py:1071]: Closing to pull from working branch.

**Pull request closed without merging**